voice/voicegateway/commands.go

```diff
@@ -105,11 +105,12 @@ const (
 )
 
 // OPCode 5
-// https://discord.com/developers/docs/topics/voice-connections#speaking-example-speaking-payload
+// https://discordapp.com/developers/docs/topics/voice-connections#speaking-example-speaking-payload
```

Comment:
Why?

Reply:
You know, I had it open on v1 and accidentally noticed the wrong branch, probably copied the wrong url over. I'll fix it